com.unity.shadergraph/CHANGELOG.md

```diff
@@ -13,23 +13,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Fixed an issue where nodes with ports on one side would appear incorrectly on creation [1262050]
 
-## [10.3.0] - 2020-11-06
-
-### Added
-
-### Changed
-
-### Fixed
-- Addressed C# warning 0649 generated by unassigned structure members
```

Comment:
Why was this removed?

Reply:
Ah, because there are two 10.3 sections.. I just merged them..

Reply:
I expect when I merge latest it will get rid of this change...